lib/twitter-ads/utils.rb

```diff
@@ -70,6 +70,18 @@ def deprecated(name, opts = {})
         warn message
       end
 
+      def extract_response_headers(headers)
+        values = {}
+        # only get "X-${name}" custom response headers
+        headers.each { |key, value|
+          if key =~ /^x-/
+            values[key.gsub(/^x-/, '').tr('-', '_')] = \
```

Comment:
I don't know if there's an simpler way to write these two lines, it's a little bit obfuscated but in a Ruby-ish way I guess it's ok....

Reply:
rubocop almost forced me to write this way so should be okay...